pkg/controller/operator.go

```diff
@@ -123,7 +126,13 @@ func (s *Support) Run(ctx context.Context, controller *controllercmd.ControllerC
 		"clusterconfig": clusterConfigGatherer,
 	})
 	statusReporter.AddSources(periodic.Sources()...)
-	go periodic.Run(4, ctx.Done())
+
+	// check we can read IO container status and we are not in crash loop
+	err = wait.PollImmediate(5*time.Second, 1*time.Minute, isRunning(ctx, gatherKubeConfig))
+	if err != nil {
+		initialDelay = wait.Jitter(5*time.Minute, 1)
```

Comment:
We should and some info log here. This will also delay every local run since we usually don't have the env vars (https://github.com/openshift/insights-operator/pull/270/files#diff-52732d3ec39ef3de250ba276b2f85c749aef6e8f9e34cd6066abaa5f165bdd1aR176) set.

Reply:
So it is really better to use someting lik  `wait.Jitter(s.Controller.Interval/12, 1)`